src/SemanticTasksMailer.php

```diff
@@ -54,15 +61,49 @@ class SemanticTasksMailer {
 	 * @throws MWException
 	 */
 	public static function mailAssigneesUpdatedTask( Assignees $assignees, WikiPage $article, User $current_user, $text,
-			$summary, $minoredit, $watchthis, $sectionanchor, $flags ) {
+			$summary, $minoredit, $watchthis, $sectionanchor, $flags, $revision ) {
 		if ( $minoredit ) {
 			return true;
 		}
 		$status = ST_UPDATE;
 		if ( ( $flags & EDIT_NEW ) && !$article->getTitle()->isTalkPage() ) {
 			$status = ST_NEWTASK;
 		}
-		return self::mailAssignees( $article, $text, $current_user, $status, $assignees );
+		$assignedToParserOutput = self::getAssignedUserFromParserOutput($article, $revision, $current_user);
+
+		return self::mailAssignees( $article, $text, $current_user, $status, $assignees, $assignedToParserOutput );
+	}
+
+	// todo: this could replace Assignees->getAssignees(...).
+	private static function getAssignedUserFromParserOutput(WikiPage $article, $revision, User $current_user) {
+		$applicationFactory = ApplicationFactory::getInstance();
```

Comment:
Probably better to call the var something like smwFactory

Reply:
Made the change.